internal/runtime/runtime_factory.go

```diff
@@ -87,21 +87,15 @@ func newSpecModifier(logger logger.Interface, cfg *config.Config, ociSpec oci.Sp
 		return nil, err
 	}
 
-	gdsModifier, err := modifier.NewGDSModifier(logger, cfg, image)
-	if err != nil {
-		return nil, err
-	}
-
-	mofedModifier, err := modifier.NewMOFEDModifier(logger, cfg, image)
+	featureModifier, err := modifier.NewFeatureGatedModifier(logger, cfg, image)
```

Comment:
As mentioned above, a simpler change for the purpose of this backport may be to add `modifier.NewGDRCopyModifier` call here instead of backporting the feature-gated modifier.